Plugins/DotNET/DotNETExports.cpp

```diff
@@ -226,8 +224,7 @@ static void StackPushString(const char* value)
     }
     else
     {
-        LOG_WARNING("Failed to push string '%s' - recursion level %i.",
-            str.m_sString, vm->m_nRecursionLevel);
+        LOG_WARNING("Failed to push string '%s' - recursion level %i.", str.m_sString, vm->m_nRecursionLevel);
```

Comment:
I prefer the multiline version because it's easier to diff side by side. Especially since log call can have a ton of arguments.

But never break the format string into multiple lines, as that kills greppability.